tests/test_all.py

```diff
@@ -433,6 +437,61 @@ def test_no_compat_packages(container):
         ).is_installed
 
 
+@pytest.mark.parametrize(
+    "container",
+    ALL_CONTAINERS,
+    indirect=True,
+)
+def test_bci_eula_is_correctly_available(container: ContainerData) -> None:
+    """Ensure that the BCI EULA exists iff it's not a LTSS container"""
```

Comment:
```suggestion
    """Ensure that the BCI EULA exists if it's not a LTSS container"""
```

Reply:
iff stands for "if and only if", so it was intentional. see https://www.dictionary.com/browse/iff